feat(curriculum): adding superhero application form (React forms) workshop (#59932)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/25-front-end-development/workshop-superhero-application-form/680900675ae3d54ee19590d1.md

---
id: 680900675ae3d54ee19590d1
title: Step 17
challengeType: 0
dashedName: step-17
---

# --description--

To finally submit the form, and a `method` attribute with a value of `post` and an `action` attribute with a value of `https://superhero-application-form.freecodecamp.org` to the form element.

> Now, having a submit button with its `type` attribute set to `submit` in the form will automatically submit the form to the URL value of that action.
> 
> You will do that in the next step.

Reviewer: ```suggestion
Now, submitting form will send it to that URL. You will do that in the next step.
```
